openassessment/templates/openassessmentblock/response/oa_response_cancelled.html

```diff
@@ -0,0 +1,52 @@
+{% extends "openassessmentblock/response/oa_response.html" %}
+{% load i18n %}
+{% load tz %}
+
+{% block list_item %}
+<li id="openassessment__response" class="openassessment__steps__step step--response ui-toggle-visibility has--error">
+{% endblock %}
+
+{% block title %}
+    <span class="step__status">
+      <span class="step__status__label">{% trans "This step's status" %}:</span>
+      <span class="step__status__value">
+        <i class="ico icon-warning-sign"></i>
+        <span class="copy">{% trans "Cancelled" %}</span>
+      </span>
+    </span>
+{% endblock %}
+
+{% block body %}
+    <div class="ui-toggle-visibility__content">
+        <div class="wrapper--step__content">
+
+            <div class="step__message message message--incomplete">
+                <h3 class="message__title">{% trans "Submission Cancelled" %}</h3>
+
+                <div class="message__content">
+                  <p>
+                    {% blocktrans with removed_datetime=workflow_cancellation.created_at|utc|date:"N j, Y H:i e" removed_by_username=workflow_cancellation.cancelled_by %}
+                        Your submission has been canceled by {{ removed_by_username }} on {{ removed_datetime }}
```

Comment:
Apparently "Cancelled" and "Canceled" are both appropriate spellings, go figure. But we should be consistent with one spelling.

Also, I think we may want to avoid having the "removed_by_username" in the message.